### ISSUE TITLE
Compile error for version 0.1.2

### ISSUE THREAD

**Issue by natboehm** [myfreeweb/systemstat#12]
Hello! I'm trying to use your crate for retrieving system uptime and received the following errors when trying to compile version `0.1.2` upon running `cargo build`:

```
   Compiling systemstat v0.1.2
error[E0432]: unresolved import `self::platform::PlatformImpl`
  --> /Users/natalie/.cargo/registry/src/github.com-1ecc6299db9ec823/systemstat-0.1.2/src/lib.rs:14:9
   |
14 | pub use self::platform::PlatformImpl as System;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `PlatformImpl` in `platform`. Did you mean to use `Platform`?

error[E0412]: cannot find type `PlatformMemory` in this scope
  --> /Users/natalie/.cargo/registry/src/github.com-1ecc6299db9ec823/systemstat-0.1.2/src/data.rs:92:26
   |
92 |     pub platform_memory: PlatformMemory,
   |                          ^^^^^^^^^^^^^^ not found in this scope

error: aborting due to 2 previous errors

error: Could not compile `systemstat`.
```
If the problem is as simple as switching out `Platform` for `PlatformImpl`, I'd be happy to submit a patch!

**Comment by myfreeweb** [comment]
Hi! This means your OS is not supported.

If it's Windows, it's been added in master (not released) but it doesn't have uptime yet. If it's macOS, there's absolutely zero support for it, sorry. Of course you can contribute support for any OS :)

**Comment by natboehm** [comment]
Okay thanks! That makes sense. I'd like say I'd have the time to do that, but it's probably not very realistic :)

**Comment by myfreeweb** [comment]
Released 0.1.3, there's not much macOS support, but uptime is one of the few supported things.
